pkg/clients/sprayproxy/sprayproxy.go

```diff
@@ -29,43 +26,43 @@ func NewSprayProxyConfig(url string, token string) (*SprayProxyConfig, error) {
 				},
 			},
 		},
-		PaCHost: pacHost,
 		Token:   token,
 	}, nil
 }
 
-func (s *SprayProxyConfig) RegisterServer() (string, error) {
-	result, err := s.sendRequest(http.MethodPost)
+func (s *SprayProxyConfig) RegisterServer(pacHost string) (string, error) {
+	bytesData := buildBodyData(pacHost)
+
+	result, err := s.sendRequest(http.MethodPost, bytes.NewReader(bytesData))
 	if err != nil {
 		return "", err
 	}
 	return result, nil
 }
 
-func (s *SprayProxyConfig) UnregisterServer() (string, error) {
-	result, err := s.sendRequest(http.MethodDelete)
+func (s *SprayProxyConfig) UnregisterServer(pacHost string) (string, error) {
+	bytesData := buildBodyData(pacHost)
+	
+	result, err := s.sendRequest(http.MethodDelete, bytes.NewReader(bytesData))
 	if err != nil {
 		return "", err
 	}
 	return result, nil
 }
 
 func (s *SprayProxyConfig) GetServers() (string, error) {
```

Comment:
I think it'd be nicer if this method returns servers in a string slice (`[]string`), wdyt

Reply:
it only unregisters one server each time, I am confused why this method  returns servers in a string slice ([]string)